gui-lite.lua

```diff
@@ -66,6 +66,12 @@ function flib_gui.add(parent, def, elems)
       if def.name then
         elems[def.name] = elem
       end
+      if style_mods.column_alignments then
+        for ikey, ivalue in ipairs(style_mods.column_alignments) do
+          elem.style.column_alignments[ikey] = ivalue
+        end
+        style_mods.column_alignments = nil
```

Comment:
Please restore this later, or better yet,  ignore this key in the following loop.